docs: explain how Rootstock's secured hashrate is calculated (#527)
rsksmart/devportal: rif/identity/rlogin/integrate/index.md

## rLogin - Integrate

{% include rif-id/rlogin-integrate.html %}

### Basic implementation:

To add rLogin to a project, first install the dependency:

```bash

> yarn add rLogin --save
> // or
> npm install rLogin --save

Reviewer: ```suggestion
yarn add rLogin --save
# or
npm install rLogin --save
```

In shell we should use `#` for comments

```

> In your application, create a rLogin instance. Make sure this instance is created outside of a render function as only one instance should be present at a time. Set the supported chain ids. In the example below, it will connect with RSK Mainnet and RSK Testnet. rLogin has been tested with RKS (mainnet and testent) and Ethereum (mainnet and testnet), however, it should work with any chain Id.

Reviewer: ```suggestion
In your application, create a rLogin instance. Make sure this instance is created outside of a render function as only one instance should be present at a time. Set the supported chain IDs. In the example below, it will connect with RSK Mainnet and RSK Testnet. rLogin has been tested with RSK (Mainnet and Testnet) and Ethereum (Mainnet and Testnet), however, it should work with any chain ID.
```

```javascript
import RLogin from '@rsksmart/rlogin'
...

> const rLogin = new RLogin({

Reviewer 2: Let's put the new interface

Reviewer 3: Sorry, I don't understand. The response from rLogin is below and does include the new interface.

Reviewer 2: Sorry you are right

  cacheProvider: true,
  providerOptions: {
    injected: {}
  },
  supportedChains: [30, 31]
})
```

> Next, initiate the connect() function. This will cause the pop up to display and allow the user to connect with their provider. rLogin will provide an object with the web3 provider, the datavault (if connected, [see the permissioned example app](https://github.com/rsksmart/rlogin-sample-apps/blob/main/permissioned-app/frontend-app/src/App.js#L63-L92)), and a disconnect/logout function.

Reviewer: ```suggestion
Next, initiate the `connect()` function. This will cause the pop up to display and allow the user to connect with their provider. rLogin will provide an object with the Web3 Provider, the data vault (if connected, [see the permissioned example app](https://github.com/rsksmart/rlogin-sample-apps/blob/main/permissioned-app/frontend-app/src/App.js#L63-L92)), and a disconnect/logout function.
```

Note that you have a link to source with line numbers against a branch, so further commits to this branch (`main`) may cause those line numbers to drift. Consider using a specific hash instead of a branch name. In addition, this would make maintenance easier because we can definitively know if the docs are up to date by analysing the commits since this particular hash.

```javascript
rLogin.connect()
  .then(response => {
    const provider = response.provider
    const dataVault = response.dataVault
    const disconnect = response.disconnect
  })
```

### Disconnect:

> To disconnect from the provider and clean up the localStorage and cookies set, use the disconnect function provided by the rLogin response. You may also have to clean up variables set in your app.

Reviewer: ```suggestion
To disconnect from the provider and clean up the `localStorage` and cookies set, use the disconnect function provided by the rLogin response. You may also have to clean up variables set in your app.
```
